# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nvidia-texture-tools/hotfix-libdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nvidia-texture-tools-2.1.2/src/nvcore/CMakeLists.txt.vanilla	2025-04-28 21:05:21.381890529 +0300
+++ nvidia-texture-tools-2.1.2/src/nvcore/CMakeLists.txt	2025-04-28 21:06:00.502000000 +0300
@@ -46,5 +46,5 @@
 
 INSTALL(TARGETS nvcore
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static)
+    LIBRARY DESTINATION lib${LIB_SUFFIX}
+    ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- nvidia-texture-tools-2.1.2/src/nvimage/CMakeLists.txt.vanilla	2025-04-28 21:05:21.384900559 +0300
+++ nvidia-texture-tools-2.1.2/src/nvimage/CMakeLists.txt	2025-04-28 21:06:16.789000000 +0300
@@ -58,6 +58,6 @@
 
 INSTALL(TARGETS nvimage
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static)
+    LIBRARY DESTINATION lib${LIB_SUFFIX}
+    ARCHIVE DESTINATION lib${LIB_SUFFIX})
 
--- nvidia-texture-tools-2.1.2/src/nvmath/CMakeLists.txt.vanilla	2025-04-28 21:02:24.971169346 +0300
+++ nvidia-texture-tools-2.1.2/src/nvmath/CMakeLists.txt	2025-04-28 21:06:30.017000000 +0300
@@ -14,11 +14,15 @@
 
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 
-ADD_LIBRARY(nvmath ${MATH_SRCS})
+IF(NVMATH_SHARED)
+	ADD_LIBRARY(nvmath SHARED ${MATH_SRCS})
+ELSE(NVMATH_SHARED)
+	ADD_LIBRARY(nvmath ${MATH_SRCS})
+ENDIF(NVMATH_SHARED)
 
 TARGET_LINK_LIBRARIES(nvmath ${LIBS} nvcore)
 
 INSTALL(TARGETS nvmath
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static)
+    LIBRARY DESTINATION lib${LIB_SUFFIX}
+    ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- nvidia-texture-tools-2.1.2/src/nvthread/CMakeLists.txt.vanilla	2025-04-28 21:02:06.530193913 +0300
+++ nvidia-texture-tools-2.1.2/src/nvthread/CMakeLists.txt	2025-04-28 21:06:53.860000000 +0300
@@ -24,5 +24,5 @@
 
 INSTALL(TARGETS nvthread
 	RUNTIME DESTINATION bin
-	LIBRARY DESTINATION lib
-	ARCHIVE DESTINATION lib/static)
+	LIBRARY DESTINATION lib${LIB_SUFFIX}
+	ARCHIVE DESTINATION lib${LIB_SUFFIX})
--- nvidia-texture-tools-2.1.2/src/nvtt/CMakeLists.txt.vanilla	2025-04-28 21:04:28.537505780 +0300
+++ nvidia-texture-tools-2.1.2/src/nvtt/CMakeLists.txt	2025-04-28 21:07:06.551000000 +0300
@@ -57,8 +57,8 @@
 
 INSTALL(TARGETS nvtt 
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static)
+    LIBRARY DESTINATION lib${LIB_SUFFIX}
+    ARCHIVE DESTINATION lib${LIB_SUFFIX})
 
 INSTALL(FILES nvtt.h nvtt_wrapper.h DESTINATION include/nvtt)
 
